scripts/restart_ecs

```diff
@@ -3,6 +3,24 @@
 # Restart ECS service
 #
 
+healthcheck_url="https://${APP_ENV}.easi.cms.gov/api/v1/healthcheck"
+healthcheck_timeout="180"
+
 cluster_name="${APP_ENV}-easi-app"
 service_name="easi-app"
 ( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment )
+
+# wait for the new service to come up
+echo -n "Waiting for health check version to match new deployment." 1>&2
+until [[ "$(jq --raw-output --monochrome-output .version < <(curl --silent --show-error --location "$healthcheck_url"))" == "$CIRCLE_SHA1" ]] ; do
```

Comment:
Have you tested this with a case where the output doesn't match the expected commit hash? It'd be nice to see what the output from the rest of the script looks like in that case.

Reply:
Locally, yes.

```
$ cat healthcheck
#!/usr/bin/env bash
#
APP_ENV="dev"

healthcheck_url="https://${APP_ENV}.easi.cms.gov/api/v1/healthcheck"
healthcheck_timeout="30"


CIRCLE_SHA1="wrong commit hash"

# wait for the new service to come up
echo -n "Waiting for health check version to match new deployment." 1>&2
until [[ "$(jq --raw-output --monochrome-output .version < <(curl --silent --show-error --location "$healthcheck_url"))" == "$CIRCLE_SHA1" ]] ; do
  if ((++time > healthcheck_timeout)) ; then
    # we timed out
    echo ""
    echo "FATAL: Timed out waiting." 1>&2
    exit 1
  else
    [[ $((time % 10)) -eq 0 ]] && echo -n "."
    sleep 1
  fi
done
echo ""
$ bash healthcheck
Waiting for health check version to match new deployment....
FATAL: Timed out waiting.
$
```